lab_1/main.py

```diff
@@ -12,7 +12,25 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    if text and type(text) == str:
```

Comment:
Usually, there are checks for incoming arguments and body function - separated.
And body function on the same level of indentation as checks.